immuni_analytics/helpers/date_utils.py

```diff
@@ -0,0 +1,28 @@
+#  Copyright (C) 2020 Presidenza del Consiglio dei Ministri.
+#  Please refer to the AUTHORS file for more information.
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Affero General Public License as
+#  published by the Free Software Foundation, either version 3 of the
+#  License, or (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  GNU Affero General Public License for more details.
+#  You should have received a copy of the GNU Affero General Public License
+#  along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+from datetime import date, timedelta
+
+
+def current_month() -> date:
+    """
+    Returns a date object representing the first day of the current month
+    """
+    return date.today().replace(day=1)
+
+
+def next_month() -> date:
+    """
+    Returns a date object representing the first day of the next month
+    """
+    return (current_month() + timedelta(days=31)).replace(day=1)
```

Comment:
well, I guess it may epically fail in some cases... if it is the end of January, it will jump to march :)

Reply:
🤔 why? `current_month` will return the date corresponding to the first day of the current month, so even if we're on the 31st of January, we will skip 31 days from the first of january (which would be the 1st of February), and then replace (uselessly in this case) the day with 1.

Reply:
oh you are right, it is adding 31 to the first day of the month, not the current date :D